app/renderer/components/Updater/FoundUpdate.js

```diff
@@ -6,18 +6,18 @@ import UpdaterStyles from './Updater.css';
 export default class FoundUpdate extends Component {
 
   render () {
-    const {hasUpdateAvailable, updateInfo} = this.props;
+    const {hasUpdateAvailable, updateInfo, releaseNotes} = this.props;
 
     if (!hasUpdateAvailable) {
       return null;
     }
 
-    const {releaseDate, releaseNotes, version} = updateInfo;
+    const {releaseDate, version} = updateInfo;
 
     return <div className={UpdaterStyles['found-updates-container']}>
       <h3>A new version of Appium Desktop is available: <span className={UpdaterStyles['release-info']}><span>{version}</span> released <span>{releaseDate}</span></span></h3>
       <div>
-        <textarea defaultValue={releaseNotes}></textarea>
+        {releaseNotes && <textarea defaultValue={releaseNotes}></textarea>}
```

Comment:
maybe if we don't have release notes, add some text to the effect of: "Release notes could not be found. To view changes since the last version, visit https://github.com/appium/appium-desktop/releases". That way people have a link to it?